satellite/launch notes

```diff
@@ -0,0 +1,4 @@
+1. point_stamped_and_odom_to_navsatfix
+2.rviz_satellite demo.launch
+3. should be it
+4. rviz open the config
```

Comment:
these instructions should just go into the readme (I also think they're inaccurate, as the file is launch.launch)